drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -103,8 +104,30 @@ protected void setInstanceHeader(ScenarioSimulationContext context, ScenarioGrid
         final FactIdentifier factIdentifier = setEditableHeadersAndGetFactIdentifier(context, selectedColumn, alias, fullClassName);
         setInstanceHeaderMetaData(selectedColumn, alias, factIdentifier);
         final ScenarioHeaderMetaData propertyHeaderMetaData = selectedColumn.getPropertyHeaderMetaData();
-        setPropertyMetaData(propertyHeaderMetaData, getPropertyPlaceHolder(columnIndex), false, selectedColumn, ScenarioSimulationEditorConstants.INSTANCE.defineValidType());
-        context.getModel().updateColumnInstance(columnIndex, selectedColumn);
+        final boolean isDMNScenario = Objects.equals(context.getModel().getSimulation().orElseThrow(IllegalStateException::new).getSimulationDescriptor().getType(),
```

Comment:
Background model PR will have impact on this code

Reply:
@danielezonca This is a good spot, in my opinion this is NOT the only point where we will have conflicts with Background. Another point is on TestTools, in my opinion. I'll think if there is an alternative to better resolve these conflicts.

Reply:
@danielezonca @yesamer 
inside background refactoring the type is defined in the settings, it won't be in the simulationDescriptor anymore -